### ISSUE TITLE
iOS 14 crashes

### ISSUE THREAD

**Issue by gabhab** [einsteinx2/iSub3#14]
I'm getting a lot of crashes on iOS 14 - in particular clicking on the search field instantly crashes it every time.

**Comment by cascooscuro** [comment]
same here, searching crashes the app.

**Comment by expressIndustries** [comment]
Agree!  Crashes all the time.  Now crashing after reloading app and trying to add server.
Help!

**Comment by gstlouis** [comment]
I think iSub needs to be updated to work with the new Local Network Privacy/Permission model introduced with iOS14. The app seems to crash whenever it’s trying to communicate with a server.

**Comment by expressIndustries** [comment]
Yeh unfortunately had to uninstall.  It was unusable.  Would be thankful
for this fix and a working version.  No other apps come close to iSub for
connecting to SubSonic on mobile.

On October 23, 2020 at 11:14:11 PM, gstlouis (notifications@github.com)
wrote:

I think iSub needs to be updated to work with the new Local Network
Privacy/Permission model introduced with iOS14. The app seems to crash
whenever it’s trying to communicate with a server.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/einsteinx2/iSub3/issues/14#issuecomment-715670360>, or
unsubscribe

**Comment by Cardboard231** [comment]
The only problem I am experiencing is that offline is crashing the app. Unfortunately to fix this I forced offline mode and now I cannot open the app to disable offline mode.

Anyway it appears IOS14 has done it's worst.
